### ISSUE TITLE
Improve CAIRR documentation

### ISSUE THREAD

**Issue by marcosmro** [metadatacenter/cedar-project#1011]
The page at http://docs.airr-community.org/en/latest/miairr/manual_miairr_ncbi.html does not currently include any directions for CEDAR (CAIRR) submission of AIRR data. John has suggested to create a 3- or 4-step summary of steps. This task needs to be done before AIRR MiniStd WG meeting on 8/9/2019.

**Comment by marcosmro** [comment]
A draft of the new documentation is available at https://github.com/marcosmro/airr-standards/blob/patch-1/docs/miairr/manual_miairr_ncbi.rst. It needs to be reviewed by @ahmadchan before publishing it.

**Comment by marcosmro** [comment]
Created a pull request with the new documentation: https://github.com/airr-community/airr-standards/pull/223. I will update the old CAIRR pipeline page (https://docs.airr-community.org/en/latest/cairr/overview.html) to point to the new documentation and to remove all the information and screenshots that are out of date.

**Comment by marcosmro** [comment]
The documentation is being restructured (https://github.com/airr-community/airr-standards/issues/199) and stored into metadata-docs branch (https://github.com/airr-community/airr-standards/tree/metadata-docs). My pull request has been accepted and merged into that branch.
